scripts/cli/src/util/init.ts

```diff
@@ -251,9 +254,31 @@ export async function sendTx(
   signer: KeyringPair,
   tx: SubmittableExtrinsic<"promise">
 ) {
-  const nonceObj = { nonce: await getNonce(signer) };
-  const result = await sendTransaction(signer, tx, nonceObj);
-  return result;
+ try {
+   const result = await sendTransaction(signer, tx, -1);
+   return result;
+ } catch (err: any) {
+   const errors = [
+     "Priority is too low",
+     "outdated",
+     "Transaction Aborted",
+     "Unknown error",
+     "settlement.UnexpectedAffirmationStatus",
+     "settlement.UnknownInstruction",
+   ];
+
+   let repeat = false;
+   for (let error of errors) {
+     err.message.includes(error) ? (repeat = true) : "";
+   }
+   
+   if (repeat) {
+     sleep(1250);
+     await sendTx(signer, tx);
+   } else {
+     throw new Error(err);
+   }
+ }
```

Comment:
I think ignoring these errors and retrying the transaction is a bad idea.

If we do allow this, then there needs to be an explanation about each type or error that is ignored.

Reply:
Understood, I would remove it, this was just what I was trying out for the tests to run in parallel but it isn't currently being used. I am running the tests synchronously using `--runInBand`  flag for the tests. Haven't found the best way to make the tests run in parallel as yet since the nonce is changed often for Alice who is used as the CDD Provider for the tests.